[2024-02-25T13:10:06.902+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2024-02-25T13:10:06.921+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2024-02-25T13:10:06.921+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2024-02-25T13:10:07.645+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): second_task> on 2022-01-03 00:00:00+00:00
[2024-02-25T13:10:07.709+0000] {standard_task_runner.py:57} INFO - Started process 1817 to run task
[2024-02-25T13:10:07.744+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-03T00:00:00+00:00', '--job-id', '483', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpx47k74g8']
[2024-02-25T13:10:07.770+0000] {standard_task_runner.py:85} INFO - Job 483: Subtask second_task
[2024-02-25T13:10:08.125+0000] {task_command.py:410} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-03T00:00:00+00:00 [running]> on host 0c40846c7775
[2024-02-25T13:10:08.774+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-03T00:00:00+00:00'
[2024-02-25T13:10:08.779+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-25T13:10:08.780+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-02-25T13:10:08.836+0000] {subprocess.py:86} INFO - Output:
[2024-02-25T13:10:08.872+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-02-25T13:10:08.885+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-25T13:10:08.995+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220103T000000, start_date=20240225T131006, end_date=20240225T131008
[2024-02-25T13:10:09.132+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-25T13:10:09.307+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
